src/views/Benchmark/index.jsx

```diff
@@ -19,6 +21,8 @@ class Benchmark extends Component {
     this.mounted = true;
     const { platform, benchmark, timeRange } = this.props;
     this.fetchData(platform, benchmark, timeRange);
+    registerTooltipPlugin();
+    Chart.defaults.global.onClick = this.handleTooltipClick;
```

Comment:
Let's handle the tooltip clicks as part of `ChartJSWrapper`.
The `Benchmark` component should not require any changes.